include/swift/SIL/SILValue.h

```diff
@@ -713,20 +722,23 @@ getOwnershipConstraint(OperandOwnership operandOwnership) {
   }
 }
 
-// Forwarding instructions have a dynamic ownership kind. Their forwarded
-// operand constraint depends on that dynamic result ownership. If the result is
-// owned, then the instruction moves owned operand to its result, ending its
-// lifetime. If the result is guaranteed value, then the instruction propagates
-// the lifetime of its borrows operand through its result.
+/// Forwarding instructions have a dynamic ownership kind. Their forwarded
```

Comment:
Saying "dynamic ownership kind" makes it sound like it isn't fixed in the IR. My thought: "changeable, albeit static ownership kind"

Reply:
Also you need to explain allowUnowned.

Reply:
I ended up duplicating the declaration's doc comment, which I think is sometimes helpful
```
/// Return the OperandOwnership for a forwarded operand when the forwarded
/// result has this ValueOwnershipKind. \p allowUnowned is true for a subset
/// of forwarding operations that are allowed to propagate Unowned values.
///
/// The ownership of a forwarded value is derived from the forwarding
/// instruction's constant ownership attribute. If the result is owned, then the
/// instruction moves owned operand to its result, ending its lifetime. If the
/// result is guaranteed value, then the instruction propagates the lifetime of
/// its borrows operand through its result.
```